Makefile

```diff
@@ -21,9 +27,6 @@ SUBDIR+=	hatponboard-lib
 SUBDIR+=	head-action
 SUBDIR+=	hey5-description
 SUBDIR+=	hpp-affordance
-SUBDIR+=	hpp-benchmark
-SUBDIR+=	hrp2-14-description
-SUBDIR+=	hrp2-dev
```

Comment:
This still shouldn't be removed. Nothing should be removed or commented in this file.